server/app/src/androidTest/java/sh/calaba/instrumentationbackend/query/ast/UIQueryASTWith.java

```diff
@@ -289,6 +291,18 @@ private boolean isMarked(Object o, Object expectedValue) {
 			return true;
 		}
 
+		// Workaround for Android 2.3.x: retrieve text and hint manually to resolve issue with AppCompat elements.
+		if (Build.VERSION.SDK_INT <= 10 && view instanceof TextView) {
```

Comment:
If its an issue with AppCompatTextView then i would expect the instanceof to be using  that? Or is it to avoid bringing in a dependency?

Reply:
Yes, I don't want to add android.support.v7.* All supported appcompat elements are instances of TextView.